### PR TITLE
Add cache module structure

### PR DESCRIPTION
**Pull request by jreynard-code**
Add cache module structure
Add 3 endpoints sample in cache module

### CODE REVIEW

File: twincache/src/main/openapi/twincache.yaml [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+      responses:
+        '200':
+          description: Successful response
+          content:
+            application/json:
+              schema:
+                type: string
+            application/yaml:
+              schema:
+                type: string
```
Comment (vcarluer) [inline twincache/src/main/openapi/twincache.yaml:35]: Needs to have a response data structure here:

importRequestId (at least)
+ add an endpoint to get importStatus from importRequestId

File: twincache/src/main/openapi/twincache.yaml [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+- name: twincache
+  description: Twin Cache Management
+
+paths:
+  /massimport:
+    post:
+      operationId: mass_import
+      tags:
+        - twincache
+      description: Launch a mass import job
```
Comment (vcarluer) [inline twincache/src/main/openapi/twincache.yaml:25]: Needs to have a Body data structure here:

graphInstanceId
storageSource (need to groom if it is possible to have an agnostic source here without Azure adherence)

File: twincache/src/main/openapi/twincache.yaml [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+                type: string
+            application/yaml:
+              schema:
+                type: string
+  /query:
+    post:
+      operationId: cache_query
+      tags:
+        - twincache
+      description: Run a query on cache
```
Comment (vcarluer) [inline twincache/src/main/openapi/twincache.yaml:41]: Needs to have a body data structure here:

* graphInstanceId
* graphInstanceVersion (can be latest, by default)
* cypherQuery

File: twincache/src/main/openapi/twincache.yaml [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+              schema:
+                type: string
+  /delete:
+    delete:
+      operationId: mass_delete
+      tags:
+        - twincache
+      description: Launch a mass delete job
+      responses:
+        '200':
```
Comment (vcarluer) [inline twincache/src/main/openapi/twincache.yaml:59]: 204

File: twincache/src/main/openapi/twincache.yaml [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+
+paths:
+  /massimport:
+    post:
+      operationId: mass_import
+      tags:
+        - twincache
+      description: Launch a mass import job
+      responses:
+        '200':
```
Comment (vcarluer) [inline twincache/src/main/openapi/twincache.yaml:27]: 202

**Review by vcarluer — CHANGES_REQUESTED** [review]
twincache module => twingraph module
cache => graph in code

**Review by icreated — APPROVED** [review]
Ok

**Pull request merged**